<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageLoggerClass</class>
 <widget class="QDialog" name="MessageLoggerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Message Logger</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="logTextEdit"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="clearPushButton">
         <property name="toolTip">
          <string>Delete Messages...</string>
         </property>
         <property name="whatsThis">
          <string>Delete Messages...</string>
         </property>
         <property name="text">
          <string>Clear Logs</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="savePushButton">
         <property name="toolTip">
          <string>Save Messages...</string>
         </property>
         <property name="whatsThis">
          <string>Save Messages...</string>
         </property>
         <property name="text">
          <string>Save Logs</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="closePushButton">
         <property name="toolTip">
          <string>Close Message Logger</string>
         </property>
         <property name="whatsThis">
          <string>Close Message Logger</string>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>savePushButton</tabstop>
  <tabstop>clearPushButton</tabstop>
 </tabstops>
 <includes>
  <include location="local">stdio.h</include>
  <include location="local">stdarg.h</include>
  <include location="local">q3filedialog.h</include>
  <include location="local">qfile.h</include>
 </includes>
 <resources/>
 <connections/>
 <slots>
  <slot>testingSignals()</slot>
 </slots>
</ui>
